osu.Game/Online/API/Requests/GetSystemTitleRequest.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Online.API.Requests.Responses;
+
+namespace osu.Game.Online.API.Requests
+{
+    public class GetSystemTitleRequest : OsuJsonWebRequest<APISystemTitle>
+    {
+        public GetSystemTitleRequest()
+            : base(@"https://assets.ppy.sh/lazer-status.json")
```

Comment:
Astute readers will inevitably point out that this (a) doesn't exist yet, (b) is not an API call.

I agreed upon doing this in this way with @peppy before.